index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>portfolio</title>
+  </head>
+  <body>
+    <header class="nav">
+      <h3>john doe</h3>
+      <i class="fa-solid fa-bars"></i>
+    </header>
+    <main class="main">
```

Comment:
- [ ] Kindly make this a section tag in order to be semantic in nature. It's good to use the section tag for key sections of your website. For this one, it's meant to be the headline section and you are required to use a `section` tag to wrap all the contents in that section. Please fix that 👍